tests/crates/enarx_exec_tests/src/bin/sgx_attestation.rs

```diff
@@ -111,6 +122,10 @@ fn main() -> std::io::Result<()> {
 
     assert!(matches!(tech, TeeTech::Sgx));
 
+    let evidence = SgxEvidence::from_der(&buffer)
+        .map_err(|e| Error::new(ErrorKind::Other, format!("SGX evidence to DER error {}", e)))?;
+    let buffer = evidence.quote;
+
     let bytes = &buffer[MRSIGNER_START..][..32];
     let out = hex::encode(bytes);
     println!("MRSIGNER = {}", out);
```

Comment:
@haraldh This works:
* Decoding from ASN1
* Getting `MRSIGNER`

This doesn't work:
* Getting the quote sig start: `let bytes = &buffer[QUOTE_SIG_START..len];`. `QUOTE_SIG_START` = 436, but this seems to be out of range.
```
MRSIGNER = 0b33f6e61130793039bcda31ca4b316c744f983ff0a72a2035f496b8eef29106
thread 'main' panicked at 'range end index 7869 out of range for slice of length 4600', tests/crates/enarx_exec_tests/src/bin/sgx_attestation.rs:133:18
```

Thoughts?

Reply:
please! simple fix... as the error says `line 133`...

```rust
    let bytes = &buffer[QUOTE_SIG_START..];
```

`len` is not the length of the quote anymore.. it's the overall length.

Reply:
Super!